cogs/public.py

```diff
@@ -1,24 +1,31 @@
 import discord
 from discord.ext import commands
 
-
 class Public(commands.Cog):
     def __init__(self, bot: commands.Bot) -> None:
         self.bot = bot
     
     @commands.hybrid_command(aliases=['av'])
-    async def avatar(self, ctx: commands.Context):
+    async def avatar(self, ctx: commands.Context, user: discord.User = None):
```

Comment:
Might be worth adding an option to retrieve (if available) a user's guild-specific avatar

Reply:
This command has `discord.User` as its argument - does it throw any errors when a user not in the guild is provided?